tests/Rector/Dynamic/ArgumentRector/config.yml

```diff
@@ -5,30 +5,27 @@ services:
             -
                 class: 'Symfony\Component\DependencyInjection\ContainerBuilder'
                 method: 'compile'
-                type: 'changed'
                 position: 0
-                default_value: false
+                replacement_value: false
 
             # added default value
             -
                 class: 'Symfony\Component\DependencyInjection\ContainerBuilder'
                 method: 'addCompilerPass'
-                type: 'changed'
                 position: 2
-                default_value: 0
+                replacement_value: 0
 
             # remove argument
             -
                 class: 'Doctrine\ORM\Persisters\Entity\AbstractEntityInheritancePersister'
                 method: 'getSelectJoinColumnSQL'
                 position: 4
-                type: 'removed'
+                replacement_value: ~
```

Comment:
Could you also add test for `replacement_value: null`?

Reply:
Isn't tilde(~) equals to null in yml?

Reply:
I don't know. How would `null` value be added then?

Reply:
Damn:D